web_src/css/review.css

```diff
@@ -290,3 +290,9 @@
   width: 100%;
   height: 8px;
 }
+
+@media (max-width: 480px) {
+  #viewed-files-summary-container {
+    display: none !important;
+  }
```

Comment:
Could it just reuse `not-mobile` instead of introducing new styles?

Reply:
It has a different pixel width:

https://github.com/go-gitea/gitea/blob/ae396ac7c02fdae65d5ea0cd5aafe5d3ca7c2ee0/web_src/css/base.css#L1285-L1290

But we could introduce a new type of helper classes for this: `hide-480`, meaning "hide below 480px width".

Reply:
> It has a different pixel width:

Yup, it is a different pixel width, but I guess `(max-width: 767.98px)` also works well for this case?

Reply:
If 767.98px works, use `.not-mobile`, otherwise make the helper as I suggested and rename `not-mobile` to `hide-768` or `hide-mobile`.

Reply:
I'll add the not-mobile class. 768px is ok.